tests/unit/src/test/resources/expect/toplevels-scala3.expect

```diff
@@ -13,6 +14,7 @@ example/Extension.scala -> example/AbstractExtension#
 example/Extension.scala -> example/Extension$package.
 example/FooEnum.scala -> example/FooEnum#
 example/FooEnum.scala -> example/FooEnum.
+example/FooEnum.scala -> example/FooEnum.
```

Comment:
Emitting companion object for enum leads to the same symbol being emitted twice if the the companion is create explicitly. Can that be problematic?  @tgodzik ?

Reply:
For toplevels it shouldn't since we use that only for figuring out where the file containing the definition is